### DISCUSSION TITLE
How to load part of a dataset with specified range of num_nodes?

### DISCUSSION THREAD

**Discussion by CEC2023-FLBONAS** [pyg-team/pytorch_geometric#7218] [Q&A]
At the first time of loading a dataset, how to select only the graphs with a specified range nodes? 
Assume that the data in the raw dataset is not in the order of the number of nodes.
Many thanks!

**Comment by wsad1** [comment]
You could pass [pre_filter](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.data.Dataset.html#torch_geometric.data.Dataset) argument to your `Dataset` class, to filter out graphs where num_nodes don't fall in a specified range.
